File: leetcode/day11/golang/main.go
package main

import (
	"fmt"

	"jonahnzhang.leetcode.com/leetcode/day11/golang/pk"
)

func main() {

	target := 6
	nums := []int{3, 2, 4}
	res := pk.TwoSum(nums, target)
	fmt.Printf("res: %v \n", res)

}

File: leetcode/day11/cpp/main.cpp
#include <iostream>
#include "so.h"

int main()
{
    Solution so;

    std::vector<int> nums = {3, 2, 4};
    int target = 6;

    for (int i : so.twoSum(nums, target))
    {
        std::cout << i << std::endl;
    }
}